### PR TITLE
[INTERNAL] Align with @ui5/logger changes to named exports

### PR DESCRIPTION
**Pull request by RandomByte**
See https://github.com/SAP/ui5-logger/pull/371

### CODE REVIEW

File: test/lib/cli/base.js
Comment (flovogt) [inline test/lib/cli/base.js:26]:
```suggestion
			isLogLevelEnabled: t.context.isLogLevelEnabledStub
```

**Review by flovogt — CHANGES_REQUESTED** [review]

File: test/lib/cli/base.js
Comment (RandomByte) [reply test/lib/cli/base.js]: Done

**Review by flovogt — COMMENTED** [review]
LGTM. Wait for logger release for final approval.

**Review by flovogt — APPROVED** [review]

**Pull request merged**
